cmds/umount/umount_linux.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
```

Comment:
I am pretty sure that is not correct. Since it ends in _linux.go it should only build on linux. But I will test on osx.